js/app.js

```diff
@@ -1,207 +1,82 @@
-      this.customersEachHour.push(customerForOneHour);
-    }
-  },
+},
+
+SalmonStore.prototype.putTheNumbersInTheBroswer = function() {
+  var tableContent = document.getElementById('storeTable');
+  this.peopleEveryHour();
+  this.totalSold();
+  var trEL = document.createElement('tr');
+  tdEL = document.createElement('td');
+  tdEL.textContent = this.name;
+  trEL.appendChild(tdEL);
+  for (var i = 0; i < timeHours.length; i++) {
+    var tdEL = document.createElement('td');
+    tdEL.textContent = this.totalCookiesSoldToday[i];
+    trEL.appendChild(tdEL);
 
-  totalCookiesSoldToday: [],
-  addedTotalCookies: 0,
-
-  totalSold: function() {
-    for (var i=0; i < timeHours.length; i++) {
-      var totalCookiesSold = this.customersEachHour[i] * this.avgCookies;
-      this.totalCookiesSoldToday.push(Math.floor(totalCookiesSold));
-      this.addedTotalCookies = this.totalCookiesSoldToday[i] + this.addedTotalCookies;
-    }
-  },
-
-  putTheNumbersInTheBroswer: function() {
-    var centerlist = document.getElementById('center');
-    this.peopleEveryHour();
-    this.totalSold();
-    for (var i = 0; i < timeHours.length; i++) {
-      var liEL = document.createElement('li');
-      liEL.textContent = timeHours[i] + ': ' + this.totalCookiesSoldToday[i] + ' cookies';
-      centerlist.appendChild(liEL);
-    }
-    liEL = document.createElement('li');
-    liEL.textContent = 'total sold: ' + this.addedTotalCookies;
-    centerlist.appendChild(liEL);
-  }
 };
 
-center.putTheNumbersInTheBroswer();
-
-var hill= {
-  name: 'Hill',
-  minCustomers: 23,
-  maxCustomers: 65,
-  avgCookies: 6.3,
-  customersEachHour: [],
-  peopleEveryHour: function() {
-    for (var i = 0; i < timeHours.length; i++) {
-      var customerForOneHour =  Math.floor(Math.random() * (this.maxCustomers - this.minCustomers + 1)+ this.minCustomers);
-      this.customersEachHour.push(customerForOneHour);
-    }
-  },
-
-  totalCookiesSoldToday: [],
-  addedTotalCookies: 0,
-
-  totalSold: function() {
-    for (var i=0; i < timeHours.length; i++) {
-      var totalCookiesSold = this.customersEachHour[i] * this.avgCookies;
-      this.totalCookiesSoldToday.push(Math.floor(totalCookiesSold));
-      this.addedTotalCookies = this.totalCookiesSoldToday[i] + this.addedTotalCookies;
-    }
-  },
+pike.putTheNumbersInTheBroswer();
 
-  putTheNumbersInTheBroswer: function() {
-    var hilllist = document.getElementById('hill');
-    this.peopleEveryHour();
-    this.totalSold();
-    for (var i = 0; i < timeHours.length; i++) {
-      var liEL = document.createElement('li');
-      liEL.textContent = timeHours[i] + ': ' + this.totalCookiesSoldToday[i] + ' cookies';
-      hilllist.appendChild(liEL);
-    }
-    liEL = document.createElement('li');
-    liEL.textContent = 'total sold: ' + this.addedTotalCookies;
-    hilllist.appendChild(liEL);
   }
+  tableContent.appendChild(trEL);
```

Comment:
You may want to move this line to be the last line in this function. It still works as is, however the logic flows better when `trEL` is appended to the DOM after all the `td` elements have been appended to the row.